playbooks/group_vars/all.yml

```diff
@@ -14,3 +14,11 @@
 
 # This is the location where the rpc repository was checked out
 rpc_repo_path: /opt/os-ansible-deployment
+
+# This is the OSAD version you are deploying rpc-extras against
```

Comment:
Is this correct? I don't think so - in the build play we are building an entirely separate repository, not including these packages in the OSAD repository.